ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/wizards/plugin/NewProjectCreationPage.java

```diff
@@ -162,6 +162,9 @@ protected void createFormatGroup(Composite container) {
 	private void updateAutomatic() {
 		boolean standardSelected = PDEUIMessages.NewProjectCreationPage_standard.equals(fOSGiCombo.getText());
 		useAutomaticMetadata.setVisible(standardSelected);
+		if (!standardSelected) {
+			useAutomaticMetadata.setSelection(false);
```

Comment:
I'm not sure it should be fixed in the UI but at the place where the value is evaluated.